VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionTopicsFragment.java

```diff
@@ -102,8 +104,12 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
             public boolean onQueryTextSubmit(String query) {
                 if (query == null || query.trim().length() == 0)
                     return false;
-
-                router.showCourseDiscussionPostsForSearchQuery(getActivity(), query, courseData);
+                Activity activity = getActivity();
+                if (activity != null) {
```

Comment:
This check is not necessary, since the Activity should always be present upon the text submission.